RELEASE-NOTES.txt

```diff
@@ -14,8 +14,9 @@
 18.8
 -----
 - [*] Menu > Payments > Collect Payment: fix the the "card reader" & "Tap To Pay" payment methods where it was no-op before. [https://github.com/woocommerce/woocommerce-ios/pull/12801]
-- [internal] Optimize API calls sent for Dashboard screen. [https://github.com/woocommerce/woocommerce-ios/pull/12775]
 - [**] Adds edit support for the Min/Max Quantities extension in product details. [https://github.com/woocommerce/woocommerce-ios/pull/12758]
+- [internal] Optimize API calls sent for Dashboard screen. [https://github.com/woocommerce/woocommerce-ios/pull/12775]
+- [internal] Enhanced Thread Safety in Core Data Management. [https://github.com/woocommerce/woocommerce-ios/pull/12805]
```

Comment:
Nit: please ensure to move this note to the latest release when it gets merged.

Reply:
Done fdabc3656f0397006785450e92d2ce1c2f40a67e but I believe I will need to move it again, I am not confident that this PR will be approved this week. 😄